### ISSUE TITLE
org-odt-node-property export not honoring manual line breaks

### ISSUE THREAD

**Issue by zenshinji** [kjambunathan/org-mode-ox-odt#95]
Properties are not exporting correctly when

`#+options: prop:t d:t p:t f:t c:t |:t \n:t`

is set. The opening colon is missing. Manual line breaks are not being honored.

For the missing colon, org-odt-node-property needs to be edited so that the format looks like

`   (format ":%s:%s"`

I am not sure how to fix the properties export preserving manual line breaks. Maybe org-odt-plain-text should handle it in the node-property export, but it does not.

Apologies for not attaching a patch for org-odt-node-property.

**Comment by zenshinji** [comment]
Exported property elements look better without the leading colon. Feel free to leave it as-is. The line breaks issue is still open.

**Comment by kjambunathan** [comment]
Please unzip the attached file.  Please execute the babel block there, before exporting the file.    If everything goes right, the attached screenshots are what you will see.  Ensure that you are running `ox-odt.el' from my repo for getting the matching results.  

[drawer-issue#95.zip](https://github.com/kjambunathan/org-mode-ox-odt/files/7027758/drawer-issue.95.zip)

[drawer.org.txt](https://github.com/kjambunathan/org-mode-ox-odt/files/7027759/drawer.org.txt)


![Screenshot from 2021-08-22 20-52-36](https://user-images.githubusercontent.com/19869864/130363337-461196cd-1474-4f69-8b7e-05b2dd743eea.png)


![Screenshot from 2021-08-22 20-54-17](https://user-images.githubusercontent.com/19869864/130363341-17786861-4e06-4f74-a877-2307f63b2cf7.png)

**Comment by kjambunathan** [comment]
Github is not formatting the elisp blocks.  So I am attaching the `elisp` portion here.  (It is exactly same as what is shared above)


```elisp
(require 'ox-odt)

(if nil			      ; flip t to nil for exporting as table
    (progn
      (setq org-odt-property-drawer-function #'org-odt-property-drawer--as-table)
      (setq org-odt-node-property-function #'org-odt-node-property--as-table))
  (progn
    (setq org-odt-property-drawer-function #'org-odt-property-drawer--plain)
    (setq org-odt-node-property-function #'org-odt-node-property--plain)))

(defun org-odt-property-drawer (property-drawer contents info)
  "Transcode a PROPERTY-DRAWER element from Org to ODT.
CONTENTS holds the contents of the drawer.  INFO is a plist
holding contextual information."
  (funcall org-odt-property-drawer-function property-drawer contents info))

(defun org-odt-node-property (node-property contents info)
  "Transcode a NODE-PROPERTY element from Org to ODT.
CONTENTS is nil.  INFO is a plist holding contextual
information."
  (funcall org-odt-node-property-function node-property contents info))

(defun org-odt-node-property--plain (node-property _contents info)
  "Transcode a NODE-PROPERTY element from Org to ODT.
CONTENTS is nil.  INFO is a plist holding contextual
information."
  (format "<text:p text:style-name=\"%s\">%s</text:p>"
	  (concat "OrgPropertiesBlock"
		  (when (org-export-last-sibling-p node-property info)
		    "LastLine"))
	  (format "%s: <text:tab/>%s"
		  (format
		   "<text:span text:style-name=\"%s\">%s</text:span>"
		   "OrgPropertyName" (org-element-property :key node-property))
		  (format
		   "<text:span text:style-name=\"%s\">%s</text:span>"
		   "OrgPropertyValue"
		   (org-odt--encode-plain-text
		    (let ((value (org-element-property :value node-property)))
		      (if value (concat " " value) "")))))))

(defun org-odt-property-drawer--plain (_property-drawer contents _info)
  "Transcode a PROPERTY-DRAWER element from Org to ODT.
CONTENTS holds the contents of the drawer.  INFO is a plist
holding contextual information."
  contents)

(defun org-odt--translate-property-drawer->table (property-drawer)
  (let* ((table-style nil)
	 ;; (table-style "GriddedTable")
	 (table-style "TableWithHeaderRowAndColumn")
	 (property-name-column-name "Property Name")
	 (property-value-column-name "Property Name")
	 (table-props (let ((props (list :type 'org)))
			(when table-style
			  (setq props (plist-put props :attr_odt
						 (list (format ":style \"%s\"" table-style)))))
			props))
	 (lispy-table
	  (org-element-map property-drawer 'node-property
	    (lambda (node-property)
	      (list (substring-no-properties (org-element-property :key node-property))
		    (substring-no-properties (org-element-property :value node-property))))))
	 (lispy-table
	  (append (list (when (or property-name-column-name
				  property-value-column-name)
			  (list property-name-column-name property-value-column-name))
			nil)
		  lispy-table))
	 (org-table
	  (apply 'org-element-adopt-elements
		 (list 'table
		       table-props)
		 (cl-loop for row in lispy-table collect
			  (cond
			   ((null row)
			    (org-element-adopt-elements
				(list 'table-row (list :type 'rule))))
			   (t
			    (apply 'org-element-adopt-elements
				   (list 'table-row (list :type 'standard))
				   (cl-loop for cell in row collect
					    (apply 'org-element-adopt-elements (list 'table-cell nil) (list cell))))))))))
    org-table))

(defun org-odt-node-property--as-table (_node-property _contents _info)
  "Transcode a NODE-PROPERTY element from Org to ODT.
CONTENTS is nil.  INFO is a plist holding contextual
information."
  (message "Using `%s'" "org-odt-node-property--as-table")
  nil)

(defun org-odt-property-drawer--as-table (property-drawer _contents info)
  "Transcode a PROPERTY-DRAWER element from Org to ODT.
CONTENTS holds the contents of the drawer.  INFO is a plist
holding contextual information."
  (message "Using `%s'" "org-odt-property-drawer--as-table")
  (org-export-data-with-backend (org-odt--translate-property-drawer->table property-drawer)
				'odt info))
```

**Comment by kjambunathan** [comment]
> Exported property elements look better without the leading colon. Feel free to leave it as-is. The line breaks issue is still open.

This issue is fixed in `ox-odt-9.3.7.324.tar`.  Please update to the newer version.

**Comment by zenshinji** [comment]
Works beautifully and as you described.

Thank you!
